### PR TITLE
Clean up JS/CSS linter issues

### PR DESCRIPTION
**Pull request by anselmbradford**
## Changes

- Remove duplicate selectors.
- Remove `'` in comments, which stylelint doesn't seem to like (we can look at why later).
- Remove typos.
- Remove `css/footer.less` and `css/header.less`
- Autofixes JS linter issues.

## How to test this PR

1. `yarn build` should pass

### CODE REVIEW

File: cfgov/unprocessed/apps/owning-a-home/js/explore-rates/highcharts-theme.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 const HIGHCHARTS_SETTINGS = {
   colors: [ '#ADDC91' ],
   style: {
-    fontFamily: '"Avenir Next", Arial, Helvetica, sans-serif',
+    fontFamily: '"Avenir Next", Arial, sans-serif',
```
Comment (anselmbradford) [inline cfgov/unprocessed/apps/owning-a-home/js/explore-rates/highcharts-theme.js:4]: Removed this for generic font list elsewhere.

File: cfgov/unprocessed/apps/paying-for-college/css/guides.less [hunk 2]
```diff
@@ -33,14 +33,6 @@
     .vspace3 {margin-bottom:30px;}
     .vspace4 {margin-bottom:40px;}
     .vspace5 {margin-bottom:50px;}
-    .font-xs,.font-s,.font-m,.font-l,.font-xl,.font-xxl {font-family:"Avenir Next", Arial, sans-serif;}
-    .font-xs {font-size:10px;}
-    .font-s {font-size:12px; line-height:18px;}
-    .font-m {font-size:14px; line-height:20px;}
-    .font-l {font-size:16px; line-height:22px;}
-    .font-xl {font-size:18px; line-height:26px;}
```
Comment (anselmbradford) [inline cfgov/unprocessed/apps/paying-for-college/css/guides.less:41]: This was the only duplicate that adjusted the line-height, so I moved it to L9

File: cfgov/unprocessed/apps/regulations3k/css/reg-navigation.less [hunk 3]
```diff
@@ -35,7 +35,7 @@
         &-left {
             width: 88%;
         }
-        &-left {
+        &-right {
```
Comment (anselmbradford) [inline cfgov/unprocessed/apps/regulations3k/css/reg-navigation.less:38]: This appears to have been a typo in https://github.com/cfpb/consumerfinance.gov/commit/e1ddf9d8ec2d4193b6de42fef4337d5cb2507d40 when migrating 
Screen Shot 2022-04-15 at 1 22 43 PM

File: cfgov/unprocessed/apps/paying-for-college/css/guides.less [hunk 4]
```diff
@@ -418,10 +406,9 @@
     .well-icon{display:none;}
     .major-sect{border-top: 1px solid #babbbd;}
     .guide-sect-header h5{color: #D83D00;}
-    p{margin:10px 12px 10px 12px;}
-    p{margin-top:0px;margin-left:0px;}
+    p{margin:10px 12px 10px 12px;margin-top:0px;margin-left:0px;}
     #take-action .font-s p{font-family: "Avenir Next", Arial, sans-serif;}
-    .loans-table {}
+    // .loans-table {}
```
Comment (wpears) [inline cfgov/unprocessed/apps/paying-for-college/css/guides.less:411]:
```suggestion
```

File: cfgov/unprocessed/css/organisms/mortgage-performance-trends.less [hunk 5]
```diff
@@ -2,9 +2,7 @@
 // This is also imported by the chart block (in chart.less)
 @import 'cfpb-chart-builder.less';
 
-.o-mp-chart {
-
-}
+// .o-mp-chart { }
```
Comment (wpears) [inline cfgov/unprocessed/css/organisms/mortgage-performance-trends.less:5]:
```suggestion
```

**Review by wpears — CHANGES_REQUESTED** [review]

**Review by wpears — APPROVED** [review]

**Pull request merged**